tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
@jkokosar, if we used Python's open command, we could specify the universal (U) flag. The gzip.open does not support the universal flag. Maybe something like this would work as well but look nicer `open(gzip.open(f, 'rb'), 'Ur')`.